output-html.inc

```diff
@@ -1,5 +1,40 @@
 <?php
 
+function check_for_gpg_fingerprint_on_mozilla_keyserver($gpg_key) {
+  $return_bool = false;
+  if (is_callable('curl_init')) {
+    $ch = curl_init();
+    $search_url = 'http://gpg.mozilla.org/pks/lookup?search=0x' . $gpg_key . '&op=vindex';
+    curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+    curl_setopt($ch,CURLOPT_URL,$search_url);
+    $output=curl_exec($ch);
+    $return_code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+    if (isset($return_code)) {
+      if ($return_code == "404"){
+        $return_bool = false;
+      } elseif ($return_code == "200"){
+        $return_bool = true;
+      }
+    }
+    curl_close($ch);
+  }
+  return $return_bool;
+
+}
+
+function output_pgpFingerprint($s) {
```

Comment:
Any chance that this could be called with untrusted input, or does the value only come from ldap entries?